src/ScottPlot/StarAxis.cs

```diff
@@ -130,11 +142,23 @@ public void Render(PlotDimensions dims, Bitmap bmp, bool lowQuality = false)
                 }
             }
 
-            if (CategoryLabels != null && ShowCategoryLabels)
+            if (CategoryImages != null && ShowCategoryImages)
```

Comment:
I'm more partial to `if (foo is not null)` rather than `if (foo != null)`, but it doesn't matter here.

`is null` is recommended in case someone overloads `Object.Equals`, but any overload which compares equal to null for non-null is probably bad to start with.

Reply:
I'll do it this way, I prefer it as well but I went with the implementation already existing in `StarAxis.cs`, thanks

Reply:
No worries, it was probably me that wrote `!= null` anyways, I just thought we might as well improve it if we're touching that line now anyways.